draft-ietf-taps-interface.md

```diff
@@ -1157,6 +1167,8 @@ such as metered or unmetered network access. If an application needs to prohibit
 metered interfaces, this should be specified via Provisioning Domain attributes
 (see {{prop-pvd}}) or another specific property.
 
+This property must not be used to qualify endpoint candidates, e.g., for link-local or multicast addresses – see {{ifspec}} for details how qualify endpoint candidates on a per-interface basis.
+
```

Comment:
is this better: /{{ifspec}} provides details about how to qualify endpoint candidates on a per-interface basis./